etc/seahorse.profile

```diff
@@ -4,23 +4,60 @@
 # Persistent local customizations
 include seahorse.local
 # Persistent global definitions
-# added by included profile
-#include globals.local
+include globals.local
+
+blacklist /tmp/.X11-unix
```

Comment:
Remov, seahorse is graphical (or am I talking bullshit?)

Reply:
Thanks guys. I'm prone to keep it in seahorse-daemon, which is CLI.